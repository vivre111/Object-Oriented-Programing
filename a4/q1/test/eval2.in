a b + done
print
eval
set a 3
set b 6
print
eval
unset a
print
unset b
print
eval
