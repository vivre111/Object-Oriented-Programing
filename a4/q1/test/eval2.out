(a + b)
(a + b)
a has no value.
(3 + 6)
9
(a + 6)
(a + b)
a has no value.
